connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ExactlyOnceWorkerSourceTaskTest.java

```diff
@@ -687,7 +687,26 @@ public void testCommitFlushCallbackFailure() throws Exception {
             callback.onCompletion(failure, null);
             return null;
         });
-        testCommitFailure(failure);
+        testCommitFailure(failure, false);
+    }
+
+    @Test
+    public void testCommitFlushAsyncCallbackFailure() throws Exception {
```

Comment:
Should we rename `testCommitFlushCallbackFailure` to `testCommitFlushSyncCallbackFailure` to match this?